bin/nexus-repositories-create

```diff
@@ -113,7 +113,7 @@ while [[ $# -gt 0 ]]; do
         if [[ $found -eq 1 ]]; then
             # Repo was not found, try to create it.
             # Don't call create on an existing repo (CASMINST-5136).
-            echo -n >&2 "DEBUG Creating ${repo_format}/${repo_type} repository: ${repo_name} ..."
+            echo -n >&2 "Creating ${repo_format}/${repo_type} repository: ${repo_name}..."
```

Comment:
IUF will not be able to recognize this message if DEBUG is removed. End space removal is fine.